### DISCUSSION TITLE
Is cdk8s actively maintained?

### DISCUSSION THREAD

**Discussion by gnupablo** [cdk8s-team/cdk8s#2086] [General]
Hi,

In our company, we're implementing cdk8s, which is a great way of programmatically defining Kubernetes manifests. However, after reviewing the repository, commit history, and discussions, it seems that the project may not be actively maintained. I also have a few questions about the API, but I'm unsure if this discussion section is the best place to seek help.

So my question is: Is cdk8s actively maintained? If not, would you recommend an alternative to it?

Thank you.

**Comment by nraooptum** [comment]
EDIT: see https://github.com/cdk8s-team/cdk8s/discussions/2052#discussioncomment-10172237

**Reply by gnupablo** [reply]
Thank you very much! By the way, I read your first message and highly recommend using CDK8s. It helped us automate many of our processes more easily by programmatically generating manifests.

**Reply by nraooptum** [reply]
No problem, and thanks for the feedback! I am curious about how others are handling the following, if you have any feedback on that as well!

https://github.com/cdk8s-team/cdk8s/discussions/2099
